specification/indices/get_data_lifecycle_stats/IndicesGetDataLifecycleStatsResponse.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { DataStreamName } from '@_types/common'
+import { integer, long } from '@_types/Numeric'
+
+export class Response {
+  body: {
+    /**
+     * The count of data streams currently being managed by the data stream lifecycle.
+     */
+    data_stream_count: integer
+    /**
+     * Information about the data streams that are managed by the data stream lifecycle.
+     */
+    data_streams: DataStreamStats
```

Comment:
```suggestion
    data_streams: DataStreamStats[]
```

[the docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/data-streams-get-lifecycle-stats.html#get-lifecycle-stats-api-response-body) say this should be an array of stats objects.